docs/guides/js-waku/light-send-receive.md

```diff
@@ -2,30 +2,56 @@
 title: Send and Receive Messages Using Light Push and Filter
 ---
 
-This guide provides detailed steps to create a light node, send messages using the [Light Push protocol](/overview/concepts/protocols#light-push), and receive messages using the [Filter protocol](/overview/concepts/protocols#filter).
+This guide provides detailed steps to start using the `@waku/sdk` package by setting up a Light Node to send messages using the [Light Push protocol](/overview/concepts/protocols#light-push), and receive messages using the [Filter protocol](/overview/concepts/protocols#filter). Check out the [installation guide](/guides/js-waku/#installation) for steps on adding `@waku/sdk` to your project.
 
 ## Create a Light Node
 
-Set up a Waku node by creating a light node, connecting to network peers with `Light Push` and `Filter` enabled, choosing a [content topic](/overview/concepts/content-topics), and creating an `encoder` and `decoder` for [message encryption](https://rfc.vac.dev/spec/26/):
+Use the `createLightNode()` function to create a `Light Node` and interact with the Waku Network:
 
 ```js
-import {
-    createLightNode,
-    waitForRemotePeer,
-    Protocols,
-    createEncoder,
-    createDecoder,
-} from "@waku/sdk";
-
-// Create and start a light node
+import { createLightNode } from "@waku/sdk";
+
+// Create and start a Light Node
 const node = await createLightNode({ defaultBootstrap: true });
 await node.start();
 
+// Use the stop() function to stop a running node
+// await node.stop();
+```
+
+:::info
+When the `defaultBootstrap` flag is set to `true`, your node will be bootstrapped using [DNS Discovery](/overview/concepts/dns-discovery). The node does not connect to any remote peer or bootstrap node if omitted.
+:::
+
+## Connect to Remote Peers
+
+Use the `waitForRemotePeer()` function to wait for the node to connect with peers on the Waku Network:
+
+```js
+import { waitForRemotePeer } from "@waku/sdk";
+
 // Wait for a successful peer connection
+await waitForRemotePeer(node);
+```
+
+The `protocols` option allows you to specify the [protocols](/overview/concepts/protocols) that the remote peers should have enabled:
```

Comment:
This is more an advacned usage of the API. by default Id expect dev to not specify protocols and let the SDK handle it.